pyproject.toml

```diff
@@ -48,6 +48,10 @@ dependencies = [
     "tzdata==2025.2",
     "urllib3==2.5.0",
     "wcwidth==0.2.13",
+    "flask",
```

Comment:
I think we've had a spearate pyproject.toml for each of the buildable images instead of using the top level one. So instead of adding here, createing a new one in slack-service would make sense.